lib/data/dashboardEvent.ts

```
@@ -77,111 +92,177 @@ export const createHumanSupportRequestEventPayload = (
-    ),
-    orderBy: desc(conversationMessages.createdAt),
-    limit: 20,
+    where: and(eq(conversations.mailboxId, mailbox.id), before ? lt(conversations.createdAt, before) : undefined),
+    orderBy: desc(conversations.createdAt),
+    limit: 50, // Get more conversations since we'll be filtering to get latest activity per conversation
   });
 
-  if (messages.length === 0) return [];
+  if (recentConversations.length === 0) return [];
+
+  const conversationIds = recentConversations.map((c) => c.id);
```

ESLint

[error] 'conversationIds' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)